.github/workflows/publish-ectd-bundle.yaml

```diff
@@ -0,0 +1,93 @@
+on:
+    workflow_dispatch:
+    push:
+      branches:
+        - main
+      paths:
+        - 'ectd_readme/README.qmd'
+        - 'adrg/_adrg.qmd'
+        - 'cover-letter/cover-letter.qmd'
+        - 'pilot5-submission/**'
+        - 'renv.lock'
+
+name: Publish ECTD bundle
+  
+jobs:
+    build-deploy:
+      runs-on: ubuntu-latest
+      permissions:
+        contents: write
+        pages: write
+        id-token: write
+      steps:
+        - name: Check out repository
+          uses: actions/checkout@v4
+          with:
+            submodules: recursive
+
+        - name: Install Nix
+          uses: DeterminateSystems/nix-installer-action@main
+          with:
+            logger: pretty
+            log-directives: nix-installer=trace
+            backtrace: full
+
+        - name: Nix cache
+          uses: DeterminateSystems/magic-nix-cache-action@main
+
+        - name: Build development environment
+          run: |
+            nix-build
+  
+        - name: Render ADRG PDF (custom)
+          env:
+            QUARTO_PRINT_STACK: true
+          run: |
+            nix-shell --run 'quarto render adrg/adrg-quarto-pdf.qmd --to pdf'
+          shell: bash
+
+        - name: Publish ADRG HTML to QuartoPub
+          env:
+            QUARTO_PUB_AUTH_TOKEN: ${{ secrets.QUARTO_PUB_AUTH_TOKEN }}
+          run: |
+            nix-shell --run 'cd adrg; quarto publish --no-prompt --no-browser adrg-quarto-html.qmd'
+  
+        - name: Render ECTD README (custom)
+          if: ${{ false }}
+          env:
+            QUARTO_PRINT_STACK: true
+          run: |
+            nix-shell --run 'quarto render ectd_readme/README.qmd --to gfm'
+  
+        - name: Render cover letter (custom)
+          if: ${{ false }}
+          env:
+            QUARTO PRINT STACK: true
+          run: |
+            nix-shell --run 'quarto render cover-letter/cover-letter.qmd'
+  
+        - name: Set up object storage s3cmd cli tool
+          uses: s3-actions/s3cmd@v1.6.1
+          with:
+            provider: linode
+            region: 'us-east-1'
+            access_key: ${{ secrets.S3_ACCESS_KEY }}
+            secret_key: ${{ secrets.S3_SECRET_KEY }}
+
+        - name: Publish ADRG PDF to object storage
+          run: |
+            s3cmd put adrg/adrg-quarto-pdf.pdf --mime-type 'application/pdf' --acl-public s3://rsubmission-draft/pilot4-container-adrg.pdf
+
+        - name: Check out ECTD bundle destination repo
+          uses: actions/checkout@v4
+          with:
+            repository: RConsortium/submissions-pilot5-datasetjson-to-fda
+            path: ./submissions-pilot5-datasetjson-to-fda
+            token: ${{ secrets.API_TOKEN_GITHUB }}
+  
+        - name: Create ECTD bundle
+          run: bash ./.github/scripts/create-ectd-bundle.sh
+  
```

Comment:
Along with creating the ectd-bundle, I think it would be helpful to add it as an artifact in the pipeline to be able to link it to a release and troubleshoot if anything goes wrong: https://docs.github.com/en/actions/tutorials/store-and-share-data